sealer/cmd/build.go

```diff
@@ -74,7 +81,9 @@ func init() {
 	buildCmd.Flags().StringVarP(&buildConfig.BuildType, "mode", "m", "lite", "cluster image build type, default is lite")
 	buildCmd.Flags().StringVarP(&buildConfig.KubefileName, "kubefile", "f", "Kubefile", "kubefile filepath")
 	buildCmd.Flags().StringVarP(&buildConfig.ImageName, "imageName", "t", "", "cluster image name")
+	buildCmd.Flags().StringVarP(&buildConfig.KubeVersion, "kubeversion", "k", "", "KubeVersion is a SemVer constraint specifying the version of Kubernetes required.")
```

Comment:
Why needs this argument?

Reply:
i think user could specify the version constraint